src/Native/Unix/System.Security.Cryptography.Native/opensslshim.cpp

```diff
@@ -35,7 +47,8 @@ bool OpenLibrary()
         libssl = dlopen(soName, RTLD_LAZY);
     }
 
-    if (libssl == nullptr)
+    // Prefer OpenSSL 1.0.x
+    if (libssl == NULL)
```

Comment:
Seems weird that this one uses `NULL` when the rest use `nullptr`

Reply:
Fixed now.